toy_app/app/views/users/index.html.erb

```diff
@@ -0,0 +1,30 @@
+<h1>Listing users</h1>
+<% flash.each do |mess, msg| %>
+  <div style="color: red; font-weight: bold;">
+    <%= msg %>
+  </div>
+<% end %>
```

Comment:
These lines code is duplicate. You can add a partial file with this content and render it in the body of file `app/views/layoust/application.html.erb`